config.sh

```diff
@@ -7,44 +7,39 @@ function build_wheel {
     local lib_plat=$PLAT
     if [ -n "$IS_OSX" ]; then
         install_gfortran
+    else
+        # For manylinux2010 builds with manylinux1 openblas builds
+        $use_sudo yum install -y libgfortran-4.4.7
```

Comment:
This is needed to use the manylinux2010 openblas builds on the manylinux1 wheel builds. `libfortran` will be statically linked (uses the `*.a` file) so the wheels are still valid.

Reply:
Sounds like the comment is backward. Maybe "Needed for manylinux1 wheels using libraries built for manylinux2010."